fix(#2069): Add Render Hook for Automatic Link Checking for Markdown Links (#2094)
medic/cht-docs: README.md

Any users who experience errors running `hugo server`, please see our [Troubleshooting guide](./troubleshooting.md).

## Link Checking

> We have automatic link validation built into Hugo that checks all internal markdown links at build time, plus two optional external link checking tools:

Reviewer: ```suggestion
This repository has automatic link validation built into Hugo that checks all internal markdown links at build time, plus two optional external link checking tools:
```
